.freeCodeCamp/tooling/server.js

```diff
@@ -127,7 +127,7 @@ const handle = {
 const wss = new WebSocketServer({ server });
 
 wss.on('connection', function connection(ws) {
-  hotReload(ws, freeCodeCampConfig.hotReload.ignore);
+  hotReload(ws, freeCodeCampConfig?.hotReload?.ignore);
```

Comment:
`freeCodeCampConfig` must always exist. Otherwise we have bigger issues.

```suggestion
  hotReload(ws, freeCodeCampConfig.hotReload?.ignore);
```